Examples/en/TypeScript/Primitives/Union and Intersection Types.ts

```diff
@@ -0,0 +1,59 @@
+// Type unions are a way of declaring that an object
+// could be more than one type.
+
+type StringOrNumber = string | number;
+type ProcessStates = "open" | "closed";
+type OddNumbersUnderTen = 1 | 3 | 5 | 7 | 9;
+type AMessyUnion = "hello" | 156 | { error: true };
+
+// If the use of "open" and "closed" vs string is
+// new to you, check out: example:literals
+
+// We can mix different types into a union, and
+// what we're saying is that the value is one of those types.
+
+// TypeScript will then leave you to figure out how to
```

Comment:
A suggestion: linking from here to the bit on widening and narrowing (as you do below for the *inverse*) is likely to be helpful – as it is, this might feel slightly overwhelming to someone brand new to this idea!

Reply:
Added! Thanks